### ISSUE TITLE
Add User sessions

### ISSUE THREAD

**Issue by j-toscani** [j-toscani/doppelkopf#38]
**Description (five sentences max.)**
I want to be able to identify Users and persist them

**Possible implementation**
Optional and mostly for discussion.

**Acceptance Criteria**
- A user can create an account by entering a Username
- A user can login by entering an existing Username
- The User stays logged in as long as they did not logout
- Only created users can log in

**Out of Scope**
- Passwords
- Tokens
- Strategies

**Document optional improvements** 
Optional. Could lead to a better feature.

depends on #36
